gnc/sonar/include/BVWrapper.h

```diff
@@ -0,0 +1,48 @@
+#include <iostream>
+#include <iterator>
+
+#include <bvt_sdk.h>
+
+#include <opencv2/core/core.hpp> 
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <ros/package.h>
+
+namespace bv{
+	const int BW8 = 0;
```

Comment:
Also, as per the [c++ ROS style guide](http://wiki.ros.org/CppStyleGuide/), braces should not be 'cuddled' (each brace should go in it's own line), and the contents of a namespace should not be indented.